source/index.js

```diff
@@ -79,16 +79,23 @@ PostgresDialect.prototype.supports.EXCEPTION = false;
 });
 
 // [4] Fix int to string conversion
+// Added Here, PR #57
+// https://github.com/cockroachdb/sequelize-cockroachdb/pull/57/files#diff-88a99604221821c7258fe9c24fc58a4ef42242b97aeae97d6c438db3f62be6cdR71
```

Comment:
I think we can remove the comment saying added here since we can just use the git blame. Prefer not to clutter it here.